jena-arq/pom.xml

```diff
@@ -69,6 +69,21 @@
       <artifactId>jsonld-java</artifactId>
     </dependency>
 
+    <!-- For Titanium JSON-LD 1.1 -->
+    <dependency>
+      <!-- Apache License -->
+      <groupId>com.apicatalog</groupId>
+      <artifactId>titanium-json-ld</artifactId>
+      <!-- For Titanium 0.8.6, last for javax.json -->
+      <scope>provided</scope>
+    </dependency>
+    <!-- Eclipse Public License 2.0 (category-B) -->
+    <dependency>
+      <groupId>org.glassfish</groupId>
```

Comment:
would it be better here to depend on `jakarta.json:jakarta.json-api`? That way Jena depends only on the API rather than on a concrete implementation of the JEE API. A user of Jena will still need to depend on the glassfish impl, but that's already a given. For any test code, the glassfish impl would also be needed.